tests/integration/Tribe/Tickets/Commerce/PayPal/PayPalTest.php

```diff
@@ -166,17 +170,39 @@ public function it_should_generate_attendees_for_all_tickets_in_PDT_transaction(
 		$gateway->set_transaction_data( $parsed_transaction );
 		$paypal->generate_tickets();
 
-		/** @var Post_Transient $post_transient */
-		$post_transient = tribe( 'post-transient' );
-
-		$post_transient->delete( $event_1_id, \Tribe__Tickets__Tickets::ATTENDEES_CACHE );
-		$post_transient->delete( $event_2_id, \Tribe__Tickets__Tickets::ATTENDEES_CACHE );
-
-		$attendees = tribe_tickets_get_attendees( $event_1_id );
+		$attendees = $wpdb->get_col(
```

Comment:
While this isn't a test of the `tribe_tickets_get_attendees` function - swapping it out for SQL seems...wrong?
Is this a timing thing or is it because `tribe_tickets_get_attendees` is inaccurate?

Reply:
This is a WIP mainly, trying to find a way to resolve the Travis test by bypassing potential caching issues elsewhere. Once I'm done I can add more comments.

Reply:
Cool 👍